silkrpc/common/constants.hpp

```diff
@@ -32,7 +32,7 @@ constexpr const char* kOsName{
 #elif __APPLE__ || __MACH__
     "Mac OSX"
 #elif __linux__
-    "Linux"
+    "linux/go1.16.4"
```

Comment:
The reason why Silkrpc response is different from Go RPCDaemon here is because Silkrpc is not written in Go and trying to follow the Go compiler version used by RPCDaemon seems not a good idea. TBH I already suggested to remove the Go compiler version from RPCDaemon as well, so please restore the previous value

Reply:
ok